app/controllers/AuditController.scala

```diff
@@ -109,31 +112,48 @@ class AuditController @Inject() (implicit val env: Environment[User, SessionAuth
           case "Assigned" =>
             WebpageActivityTable.save(WebpageActivity(0, anonymousUser.userId.toString, ipAddress, "Visit_Audit", timestamp))
 
-            // HITs are associated with routes at the time of HIT creation
-            // Retrieve the route based on HIT ID
-            val routeId: Option[Int] = AMTRouteAssignmentTable.findRouteByHITId(hitId)
+            // Retrieve the route based on the condition that the worker has been assigned to and the associated unvisited routes
+            var cId = AMTConditionTable.assignAvailableCondition
```

Comment:
Instead of making cId a var and then doing if/else, switch this to Scala's match syntax...
```
val cId: Int = TurkerTable.getConditionIdByTurkerId(workerId) match {
    case Some(conditionId) =>
        TurkerTable.getConditionIdByTurkerId(workerId)
    case None =>
        val newCId: Int = AMTConditionTable.assignAvailableCondition
        val turker: Turker = Turker(workerId, "",newCId.getOrElse(1))
        TurkerTable.save(turker)
        newCId
}
```